scripts/calcRD.sh

```diff
@@ -12,11 +12,15 @@ count=$3
 
```

Comment:
Is there a reason why this is a separate script rather than embedded in the WDL command? It seems like it's pretty short and only called once, unless I missed something. Usually I would support modularity and functionalization, but with WDL another thing to take into account is that having separate scripts requires you to update the docker every time you change the script, so it can be easier to maintain simple bash code when it's directly in the WDL